model/src/main/java/flipkart/lego/api/entities/ServiceResponse.java

```diff
@@ -16,11 +16,8 @@
 
 package flipkart.lego.api.entities;
 
-import flipkart.lego.api.helpers.Describable;
-import flipkart.lego.api.helpers.Identifiable;
-
 /**
  * Any entity returned by a service client
  */
-public interface ServiceResponse extends Identifiable, Describable{
+public interface ServiceResponse extends Block {
```

Comment:
This class can be removed. Its unused and doesn't make sense to make upstream service pojos implement this interface and we don't have a common way to interact with a service from lego's blocks like DS, Filter, Buildable etc.